src/main/charts/confluence/values.yaml

```diff
@@ -105,28 +105,33 @@ confluence:
 
   resources:
     jvm:
+      # -- JVM memory arguments below are based on the defaults defined for the Confluence docker container, see:
+      # https://bitbucket.org/atlassian-docker/docker-atlassian-confluence-server/src/master/#markdown-header-memory-heap-size
```

Comment:
This doesn't work for me - it doesn't scroll.

I believe we would need to use https://bitbucket.org/atlassian-docker/docker-atlassian-confluence-server/src/master/README.md#markdown-header-memory-heap-size

Reply:
Hmm works for me. Let me look...

Reply:
Honestly, none of those links work for me - those don't scroll to the point, both just open the page.

Reply:
Yes auto scrolling to the actual section doesnt work for either link. I think thats an issue on BB's side.